### DISCUSSION TITLE
Single vs album

### DISCUSSION THREAD

**Discussion by joksas** [Podcastindex-org/podcast-namespace#578] [Enhancement Proposal]
Currently, when `medium` is set to `music`, the feed is interpreted as an album. But many artists upload their songs as singles, so it's not clear how the feed should be *interpreted*.

At the moment, for most singles, the title is the same for the feed and the item, but there's no indication to the players that this is a music single.

Could we somehow communicate this in the RSS feed? This could even be extended to other mediums. Suppose we had a feed-level tag `<podcast:singleItem>true</podcast:singleItem>`.
- For `music`, this would signal that the feed is for a music single instead of an album or EP.
- For `audiobook`, this could indicate that the entire book is uploaded as a single item instead one item per chapter.

Would love to hear what you all think.

**Comment by Kolomona** [comment]
I'm not opposed to this. 
My way of doing this with the current system is outlined here.

https://v4vmusic.com/post/how-bob-organizes-his-music-uploads/

**Reply by joksas** [reply]
This is very useful, thank you!

Am I understanding it correctly that the distinction would be based on whether the podcast is serial or episodic? That could work, but I'm afraid that repurposing old specs could mean that a lot of albums are misinterpreted because we'd be making implicit assumptions, instead of using new tags that are specific for this distinction.

I also don't think that creating many one-item feeds makes things complicated. With the right UI, it's easy.

**Comment by thomasrynne** [comment]
I was expecting albums to be be represented using seasons, with apps showing the text "album" instead of "season" when the medium is music

**Comment by thomasrynne** [comment]
The value of reusing tags across mediums is that it reduces the app developer effort in supporting new mediums and apps that ignore the medium tag will partially support the feature without any changes.

**Comment by kidscallmeray** [comment]
> Currently, when `medium` is set to `music`, the feed is interpreted as an album. But many artists upload their songs as singles, so it's not clear how the feed should be _interpreted_.
> 
> At the moment, for most singles, the title is the same for the feed and the item, but there's no indication to the players that this is a music single.
> 
> Could we somehow communicate this in the RSS feed? This could even be extended to other mediums. Suppose we had a feed-level tag `<podcast:singleItem>true</podcast:singleItem>`.
> 
> * For `music`, this would signal that the feed is for a music single instead of an album or EP.
> * For `audiobook`, this could indicate that the entire book is uploaded as a single item instead one item per chapter.
> 
> Would love to hear what you all think.

I support this 100%.  There must be a distinction between singles & albums in the RSS world.  I saw on a Telegram thread that some suggest putting "(single)" in the title, but that is not an aesthetic solution to many.  From a business perspective, if I were to upload "The Wall" as an album, I would want to see not only the amount of streams & downloads for the entire album, but I would also want to know how many streams "Comfortably Numb" got in comparison to the rest of the tracks.  Those are two different things.  The overwhelming majority of professionally aspiring artists are releasing singles (just like the 1950's) - they should each come with individual metrics and analytics on the back end.  For those artists that still release albums, they should be able to see the analytical difference between an album download/streem, in addition to individual songs.

**Comment by samsethi** [comment]
@joksas we 100% support this and will have it working in our UI on Monday

**Comment by Dwev** [comment]
I [commented](https://github.com/Podcastindex-org/podcast-namespace/discussions/604#discussioncomment-8405089) on another thread about how to identify different elements of the feed on different mediums, with a proposal for discussion.  I think that standardizing this is important for a consistent user experience.  It will also help identify gaps, like the "single" issue mentioned above.

> I've included the number of feeds each medium has in the index in brackets.  Course is not official (yet).  Podcasts are the total feed minus the ones with media tags defined.

| Not Defined / Podcast (4095555) | Video (59) | Music (58) | Audiobook (8) | Film (6) | Course | _Blog (0)_ | _Newsletter (0)_ |
|---|---|---|---|---|---|---|---|
| Podcast | Vodcast (!) | Album | Audiobook | Film | Course | _Blog_ | _Newsletter_ |
| Season | Season  | Disc | Volume | N/A | Module | _Year_ | _Volume_ |
| Episode | Episode | Track | Chapter | Feature | Lesson | _Entry_ | _Issue_ |
| Chapter | Chapter | Section | Section | Section | Chapter | _Topic_ | _Header_ |
